runtime/port/zos390/j9csrsi.h

```diff
@@ -27,6 +27,9 @@
 
 typedef struct j9csrsi_t j9csrsi_t;
 
+/* defined in j9csrsi_wrp.s. */
+extern void j9csrsi_wrp(struct j9csrsi_t *session);
```

Comment:
This should have a fuller comment, like other functions declared in this file.
Also suggest `extern` and `struct` be omitted for consistency with other declarations:
```c
void j9csrsi_wrp(j9csrsi_t *session);
```

Reply:
Thanks, done. I added the fuller comment description based off the explanation in the j9csrsi_wrp.s file.
Let me know if any tweaks needed to that, or it looks good.